docs/pages/foundations/color/examples.js

```diff
@@ -201,7 +201,7 @@ export default function ColorExamplesPage(): Node {
             defaultCode={`
             <Flex>
               <Flex direction="column" gap={{ column: 2, row: 0 }}>
-                <Flex gap={{ column: 0, row: 2 }}
+                <Flex gap={{ column: 0, row: 2 }}>
```

Comment:
@dangerismycat thanks for fixing these - one more reason as to why we want these examples to be flow typed. We can achieve that by using #2221 for every example /cc @AlbertCarreras

Reply:
Agreed — we're going to prioritize that migration and start converting pages soon (I hope to convert at least one this week just to get a feel for it). The lack of errors/Flow/linting on our code examples has bugged me for such a long time